common/primitives/src/messages.rs

```diff
@@ -10,6 +10,15 @@ use sp_std::{prelude::*, vec};
 #[cfg(feature = "std")]
 use utils::*;
 
+#[cfg(feature = "runtime-benchmarks")]
```

Comment:
I would create a new rs file and put all these logic inside it and loads it only under `feature = "runtime-benchmarks"` in lib.rs

like 
```
#[cfg(feature = "runtime-benchmarks")]
pub mod benchmarking_helper;
```